src/main/java/xyz/aadev/generitech/client/gui/machines/GuiPulverizer.java

```diff
@@ -75,6 +75,7 @@ public GuiPulverizer(InventoryPlayer inventoryPlayer, TileEntityPulverizer tileE
         }
     }
 
+
     @Override
     public void drawBG(int paramInt1, int paramInt2, int paramInt3, int paramInt4) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "drawBG" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)